clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/converter/manifest/KubernetesDeployManifestConverterTest.groovy

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2019 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.clouddriver.kubernetes.v2.converter.manifest
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.google.gson.Gson
+import com.netflix.spinnaker.clouddriver.kubernetes.security.KubernetesNamedAccountCredentials
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesManifest
+import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider
+import groovy.text.SimpleTemplateEngine
+import spock.lang.Specification
+
+
+class KubernetesDeployManifestConverterTest extends Specification {
+
+  def converter = new KubernetesDeployManifestConverter()
+  def inputMap = new HashMap()
+  def manifestList = new ArrayList()
+  def gsonObj = new Gson()
```

Comment:
I'd suggest just using the existing `ObjectMapper` and doing `objectMapper.readValue()` rather than introduce a new deserialization library.